src/app/components/ElectronicDelivery/ElectronicDelivery.jsx

```diff
@@ -93,10 +98,13 @@ class ElectronicDelivery extends React.Component {
   }
 
   updateIsLoadingState(status) {
-    this.setState({ isLoading: status });
+    setTimeout(
+      () => {
+        this.setState({ isLoading: status });
+      }, 500
```

Comment:
Same comment as above. We said we'd only do the delay on the search page. You do not need it here because the Hold request takes some time anyways.

Reply:
so Hold doesn't need delay. OK I will update it.

Reply:
No, it doesn't. That was something we first talked about. The request call takes a long time as is, we only need the delay when making searches since those are fast.